app/scripts/templates/complete_reset_password.mustache

```diff
@@ -46,7 +46,7 @@
 
       <div class="input-row password-row">
         <label class="label-helper"></label>
-        <input type="password" class="password" id="password" placeholder="{{#t}}New password{{/t}}" pattern=".{8,}" autofocus required {{#isPasswordAutoCompleteDisabled}}autocomplete="off"{{/isPasswordAutoCompleteDisabled}} />
+        <input type="password" class="password" id="password" placeholder="{{#t}}New password{{/t}}" pattern=".{8,}" required autofocus /> 
```

Comment:
Looks like you got a trailing space here now after `/>`. Please remove it

![](http://v14d.com/i/56eb07b7214c1.png)